### DISCUSSION TITLE
templates for creating threads

### DISCUSSION THREAD

**Discussion by bogorad** [janhq/jan#3143] [Feature Ideas]
Suppose I have a template, something like "summarize in up to 20 bullet points, ignore mentions of XXX, elaborate on mentions of YYY, formulate in the style of Bloomberg headlines". 

This template could have a "default" LLM (e.g., Claude), or it might not have one.

So, when I create a thread, I can select from a template and be up and running in no time.

Also, grouping of threads (like into folders) would be great.

**Comment by erhhung** [comment]
This is basically the same feature as priming the chat with saved custom prompts: **Jan: Customizing and saving prompts #3624**
